### ISSUE TITLE
custom sign hook should use diary_extension

### ISSUE THREAD

**Issue by gauteh** [mattn/calendar-vim#34]
```
From 6de5eaa415f5fd01ac3c77de2fbf07c798cb3317 Mon Sep 17 00:00:00 2001
From: Gaute Hope <eg@gaute.vetsj.com>
Date: Mon, 20 Aug 2018 15:42:31 +0200
Subject: [PATCH] diary: use g:diary_extension in default calendar sign hook

---
 autoload/calendar.vim | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autoload/calendar.vim b/autoload/calendar.vim
index 824f877..f369b82 100644
--- a/autoload/calendar.vim
+++ b/autoload/calendar.vim
@@ -1073,7 +1073,7 @@ endfunc
 "*   year  : year of sign
 "*****************************************************************
 function! calendar#sign(day, month, year)
-  let sfile = g:calendar_diary."/".printf("%04d", a:year)."/".printf("%02d", a:month)."/".printf("%02d", a:day).".md"
+  let sfile = g:calendar_diary."/".printf("%04d", a:year)."/".printf("%02d", a:month)."/".printf("%02d", a:day).g:calendar_diary_extension
   return filereadable(expand(sfile))
 endfunction
 
-- 
2.18.0

```

**Comment by mattn** [comment]
Could you please send me PR?

**Comment by gauteh** [comment]
mattn writes on August 21, 2018 4:39:
> Could you please send me PR?

Sure, see #35. But you could just do `git am -` with the patch I sent 
you and I wouldn't have to set up a fork for this one-line change ;)

- gaute

**Comment by gauteh** [comment]
Fixed in #35.
